src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocTypeCheck.java

```diff
@@ -145,6 +146,28 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example that allows missing comments for classes annotated with {@code @Annotation}
```

Comment:
I suggest to change order a bit. It looks like at some previous PR usual order had changed - config should go before example code, that is a common pattern, e.g. https://checkstyle.org/config_modifier.html#InterfaceMemberImpliedModifier_Examples

Previous example (I suppose you took it as reference) can be corrected as well.

Reply:
@stoyanK7 , please do us a favor and swap config and java code blocks for both examples.

Reply:
Done!
![Screenshot from 2022-07-14 09-42-55](https://user-images.githubusercontent.com/23459549/178921848-d86de6d1-79a0-4739-b867-4ed6d85c2625.png)